applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -345,13 +343,7 @@ static const struct bt_bap_unicast_server_cb unicast_server_cb = {
 	.release = lc3_release_cb,
 };
 
-#if (CONFIG_BT_AUDIO_TX)
-static void stream_sent_cb(struct bt_bap_stream *stream)
-{
-	atomic_dec(&iso_tx_pool_alloc);
-}
-#endif /* (CONFIG_BT_AUDIO_TX) */
-
+#if (CONFIG_BT_AUDIO_RX)
```

Comment:
We have been focusing on getting rid of precompile guards. Can we solve this differently?

Reply:
Unfortunately no, the callbacks are inside compile dependent code in host. So the sent callback doesn't exist unless BT_AUDIO_RX is set, similarly with recv, but for BT_AUDIO_TX

Reply:
I found 2 other places that #if could be removed